packages/flutter_test/lib/src/binding.dart

```diff
@@ -1461,6 +1503,7 @@ class LiveTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
   Future<T> runAsync<T>(
     Future<T> callback(), {
     Duration additionalTime = const Duration(milliseconds: 1000),
+    bool awaitSideEffects = false,
```

Comment:
where is this used?

Reply:
Will be removed once we agree on implementation.